server_util_test.go

```diff
@@ -78,11 +83,21 @@ func (methodsT) echo() handler {
 
 func (methodsT) traceEnabled() handler {
 	return func(ctx context.Context, args *raw.Args) (*raw.Res, error) {
-		// span := tchannel.CurrentSpan(ctx)
-		ret := byte(0)
-		// if span.TracingEnabled() {
-		// 	ret = 1
-		// }
+		ret := byte(0x00)
+
+		span := opentracing.SpanFromContext(ctx)
+		if span != nil {
+			if spanCtx, ok := span.Context().(jaeger.SpanContext); ok {
+				if spanCtx.IsDebug() {
+					ret = 0x01
+				}
+			} else {
+				ret = 0xfe
+			}
+		} else {
+			ret = 0xff
```

Comment:
these cases were helpful in isolating test errors during development. I’ll scale it back down.